src/processor/Package.ts

```diff
@@ -78,14 +79,21 @@ export class Package {
     this.removeDefaultSlicingRules();
     this.removeDateRules();
     this.combineContainsRules();
+    this.simplifyFHIROnlyRules();
   }
 
   private resolveProfileParents(processor: FHIRProcessor): void {
     for (const profile of this.profiles) {
```

Comment:
I know this is pre-existing code, but technically and extension *can* have a parent -- so we really ought to be doing this for `this.extensions` too.

Reply:
Does the existing logic for finding parents in `processor.structureDefinitions` also apply to Extensions? If that's the case, then the whole function can apply to both Profiles and Extensions.

Reply:
Yes, I think you can just change this line to be something like:
```ts
for (const def of [...this.profiles, ...this.extensions]) {
```